lib/r10k/source/git.rb

```diff
@@ -44,7 +44,7 @@ class R10K::Source::Git < R10K::Source::Base
   # @param options [Hash] An additional set of options for this source.
   #
   # @option options [Boolean] :prefix Whether to prefix the source name to the
-  #   environment directory names. Defaults to false.
+  #   environment directory names. Defaults to false. Will be a String of the prefix otherwise.
```

Comment:
Should be brought in line with the change in base.rb and have the type changed to `[Boolean,String]`.  Also, as per [my comment on that change](https://github.com/puppetlabs/r10k/pull/295/files#r24205559), this would be clearer if it read:
> If a String, the prefix name for the environment directory names. Otherwise, whether to use the source name as the prefix. Defaults to false.